native~/Runtime/src/Cesium3DTilesetImpl.cpp

```diff
@@ -82,12 +84,24 @@ void Cesium3DTilesetImpl::Update(
   }
 
 #if UNITY_EDITOR
-  // If "Update In Editor" is false, return early.
   if (UnityEngine::Application::isEditor() &&
-      !UnityEditor::EditorApplication::isPlaying() &&
-      !tileset.updateInEditor()) {
-    return;
+      !UnityEditor::EditorApplication::isPlaying()) {
+    // If "Update In Editor" is false, return early.
+    if (!tileset.updateInEditor()) {
+      return;
+    }
+
+    // If the opaque material or any of its properties have changed, recreate
+    // the tileset to reflect those changes.
+    if (tileset.opaqueMaterial() != nullptr) {
+      int32_t opaqueMaterialHash = tileset.opaqueMaterial().ComputeCRC();
+      if (_lastOpaqueMaterialHash != opaqueMaterialHash) {
+        this->DestroyTileset(tileset);
+        _lastOpaqueMaterialHash = opaqueMaterialHash;
```

Comment:
I think this needs to be set in LoadTileset, too. Otherwise it will always be destroyed on the first tick after it's first created.